modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -40,6 +40,25 @@ This function returns null if the two given parameters are equivalent, otherwise
 
 |===
 
+=== Updated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+MATCH (n) NODETACH DELETE n
+----
+a|
+Added a new keyword `NODETACH`, explicitly defining that relationships will not be detached and deleted.
+This is a mirror of the already existing keyword `DETACH`.
```

Comment:
We have used "alias" elsewhere to describe these features - would that work here instead of mirror?